scripts/stable_bucketer

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python3
```

Comment:
I think in general we've kept the `.py` suffix even if this is a one-off script.  Other general tooling that counts python code, will have a harder time if we don't have that so unless you feel really strongly about not having the extension, I think you should tack it on.

Reply:
Hmm... I do believe that the language a tool is written in is an implementation detail and that extensions on executables are clunky and not good UI design; I'll add it if there is specific reason to or if I get additional feedback that we want extensions on our scripts as standard.

Reply:
Sounds good, other scripts in this repo have it and I know that for example github search and stats around finding python files vs other files will index this differently without the extension.  I agree with you in theory but I think the information that this is python is valuable for some tooling and searchability reasons.

That said, yea, do whatever makes sense but wanted to be clear about the tradeoff.

Reply:
So I added the .py extension, but tooling choked on the Python 3.6 f-strings, so I down-ported to 3.5 which is our target version, but tooling still seems to expect .py files to be 2.7, so dropping the extension until tools can handle at least 3.5.

Reply:
Which tooling? what was the failure exactly? Can you add it as a comment or in the commit message?

Reply:
Looked through the errors, nearly all of them were for the wrong version (or just plain wrong, [like this](https://build.testeng.edx.org/job/edx-platform-quality-pipeline-pr/9133/pylint/new/source.fcdc9af3-349c-45a5-9891-7df4fc91fe93/#109) [and this](https://build.testeng.edx.org/job/edx-platform-quality-pipeline-pr/9133/pylint/new/source.f57c6a34-5179-489f-9cd9-333d9db4bcef/#120)); fixed the one issue I care to.